Sources/ManagedAppConfigLib/AppConfigPlain.swift

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
AppConfig.swift and AppConfigPlain.swift have a lot of code (and comments) in common. Is there a way we can share code for less duplication, since they are so simiar?

Reply:
I tried to figure that out myself with attempted refactors.  But I don't want to mix the `ObservableObject` conformance and `@StateObject` property wrapper for SwiftUI into the non-SwiftUI version.  Any ideas?

Reply:
Possibly adding the `DynamicProperty` and `ObservableObject` conformance, as well as the `update()` func, in a SwiftUI-specific extension?

There might be wrinkles that prevent that though... and maybe that precludes using the SwiftUI and Plain variants within the same app?

In any case I'm going to go ahead and approve this... but something to think about.

Reply:
The biggest wrinkle to that is getting the `didSet` code onto the `value` property of the Listener for SwiftUI.  There is some possibility to move the `listenTo` function somewhere shared since the `Listener.listenTo()` function is identical for the two `Listener` classes.

The rest of it is just so slightly out of sync that I couldn't find any better way to implement it.  I'm open to any changes around it to make that easier.

It only really matters on iOS 7.0-12.x and tvOS 10.2-12.x because on all newer OS versions (including on all macOS) the SwiftUI support is going to be there anyway whether or not you're actually using SwiftUI views.  So the plain old `@AppConfig` is the way to go with **any** code (AppKit, UIKit, SwiftUI) if you only need to support modern OSes.

Reply:
Maybe I should adjust the README file with some other words around that.  Something like:

> The `@AppConfig` works on any modern OS whether or not you use SwiftUI, and it does trigger SwiftUI redraws if it is used in a SwiftUI view.

Reply:
In that case, I wonder if using `available` checks would be preferable... and having the code for AppConfigPlain just be the AppConfig class for pre-SwiftUI OS's. Then when support for these older OS's rolls off, we can just lop off the support here without affecting any user's code (since users would always be using `AppConfig` as the class name).

Reply:
I don't think `@available` will let us have two types with the same names.  Because it is type conformances (`DynamicProperty`, `ObservableObject`) and a property wrapper (`@StateObject`) in the implementation differences, I don't see a way to have one named type that does NOT have those things on older OS versions.

Reply:
Ugh. I feel like there has to be a way to make this happen, but I'm not seeing it now either :(

Reply:
Hm.  I tried putting some current `@AppConfig` calls into the unit tests, and forgot about one key thing.  If the SwiftUI-ish `@AppConfig` is used outside of a SwiftUI view context it doesn't update when changes to prefs occur.z

So I was wrong previously.  The `@AppConfig` is only useful in a SwiftUI view.  `@AppConfigPlain` is useful in any code outside of a SwiftUI view, even on modern OSes.